src/app/features/post/services/post-firebase.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { FirebaseCrudService } from '@core/services/firebase/firebase-api/firebaseCrud.service';
+import { AngularFirestore } from '@angular/fire/compat/firestore';
+import { IPost } from '@core/models/Post';
+
+@Injectable({
+	providedIn: 'root'
+})
+export class PostFirebaseService extends FirebaseCrudService<
```

Comment:
PostFirebaseService already exists, if you need to managing posts in your component, create a postsService and inject PostFirebaseService inside that service